### ISSUE TITLE
[1.11.2] Render Crash

### ISSUE THREAD

**Issue by GWSheridan** [Funwayguy/BetterQuesting#188]
Using version 2.4.226 of Better Questing
An using version 2.4.132  Better Questing - Standard Expansion

Every time I open the quest book to edit quest so as I change or delete any quest MC will crash back to launcher

https://pastebin.com/b5h31gVQ

**Comment by Funwayguy** [comment]
Strange... although the crash occurs on the Item Selection Editor, Minecraft is reporting that something is triggering a render crash related to an entity not directly associated with the screen. Do you know of any such items that would affect how other things in the world render? Might also be worth checking what entities in your vicinity could be contributing to it.

**Comment by GWSheridan** [comment]
Created new world
Changed world to peaceful an killed every entity around me
Went to edit a pickaxe an crashed to launcher

https://pastebin.com/pM3GGz2B

**Comment by GreatOrator** [comment]
Had the same type of crash:

https://pastebin.com/R55Uzrpy

Happens everytime I press the letter s or n it appears, but it is in the item selection screen for icons.

**Comment by TensorTime** [comment]
I'm wondering if it's keybinding related?  I"m running into a similar crash and what I did to get around it temporarily was create a skinny dev pack version which only had the mods I was working on quests for and the problem went away.  It was only when I went to select an item for a task or reward. 

One thing I noticed, and the reason I say keypress, is it fits the evidence at least in my case.  Mods add keypreses and removing mods I may have removed the one causing a problem.  The second reason is I've noticed that certain mods which have keybinds for GUI's will cause the GUI to pop up when I'm typing in the selection editor as well as in the description for quests editor.  So it can make it impossible to type certain letters without a bit of working around the issue if they are bound like that.

It could be something is trying to render to screen during that selection process and is failing to be able to do so causing the crash.
